src/Files.App/UserControls/InnerNavigationToolbar.xaml

```diff
@@ -644,7 +644,7 @@
 
 				<AppBarButton.Flyout>
 					<Flyout contract8Present:ShouldConstrainToRootBounds="False" Placement="Bottom">
-						<StackPanel Spacing="12">
+						<StackPanel Width="392" Spacing="12">
```

Comment:
This is surely needed to set the maximum size of the Flyout.

Reply:
Alright then. If so, do we want to conteralize the TextBlock?

Reply:
We can set a max width